frontend/src/app/footer/footer.component.html

```diff
@@ -1,4 +1,4 @@
-<footer class="footer">
```

Comment:
Hmm my intuition is that semantic html tags actually help accesibility. But this is not the case?

Reply:
In general yes. In most sites, the `<body>` is layed out roughly like this:

```html
<header></header>
<main></main>
<footer></footer>
```

This is preferable to making three `<div>` elements, because this also assigns the semantic role for those elements; respectively `banner`, `main`, and `contentinfo`. Those roles also act as landmarks, so they can help users navigate the page. The issue was that I-analyzer was layed out like this:

```html
<ia-menu role="header"></ia-menu>
<main></main>
<ia-footer role="contentinfo">
	<footer></footer>
</ia-footer>
```

So we had a footer within a footer.